config/node.go

```diff
@@ -23,12 +23,11 @@ type Node struct {
 	WalletDir                 string                `json:"wallet_dir"`
 	ApiURL                    string                `json:"api_url"`
 	Bitswap                   Bitswap               `json:"bitswap"`
-	Limits                    Limits                `json:"limits"`
+	LimitConfig               rcmgr.LimitConfig     `json:"limit_config"`
```

Comment:
best you call it `limiter`, no need to add the `config` suffix

Reply:
`Limiter  rcmgr.LimitConfig     json:"limiter"`

Reply:
Limiter refers to a different type in rcmgr. so instead to keep it consistent with the existing naming i put it back to `Limits rcmgr.ScalingLimitConfig json:"limits"`